Update Azure AD SSO docs (#96)
hashicorp/terraform-docs-common: website/docs/cloud-docs/users-teams-organizations/single-sign-on/azure-ad.mdx

1. Navigate to the single sign-on page.

> 1. Edit step 2, "User Attributes & Claims."

Reviewer: ```suggestion
1. Edit the **User Attributes & Claims**.
```

Suggest updating for proper user field formatting

   We recommend naming the claim "Username", leaving the namespace blank, and sourcing something like `user.mail` or `user.userprincipalname`.

> If you namespaced any of your claims, note that the attribute name passed by Microsoft Azure AD will follow the form `<claim_namespace/claim_name>`. Consider this when setting Team and Username attribute names.

Reviewer: ```suggestion
If you namespaced any of your claims, then Microsoft Azure AD passes the attribute name using the format `<claim_namespace/claim_name>`. Consider this format when setting Team and Username attribute names.
```

Suggest rewording to be more active voice, avoid a future tense style issue, and avoid using "This" without a subject.

Author: Thanks mate, I have committed to the suggested changes.
